setup.py

```diff
@@ -542,7 +543,7 @@ def setup_package():
     np_minversion = '1.17.3'
     np_maxversion = '9.9.99'
     python_minversion = '3.8'
-    python_maxversion = '3.10'
+    python_maxversion = '3.11'
```

Comment:
this is also already correct on the release branch, and not used on `master` I don't think; perhaps harmless to leave in

Reply:
If it isn't used, should we remove it (rather than leaving an old incorrect value that isn't used)?

Reply:
No, because it is used on release branches when they are branched off `master`. I don't think it hurts to update, just doesn't affect much. Maybe makes the update for the next release a more intuitive bump perhaps.